### PR TITLE
Add user following

### PR DESCRIPTION
**Pull request by ttuan**

### CODE REVIEW

File: app/views/relationships/create.js.erb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+$("#follow_form").html("<%= escape_javascript(render('users/unfollow')) %>");
```
Comment (phamkykhoi) [inline app/views/relationships/create.js.erb]: ' ' => " "

File: app/views/relationships/destroy.js.erb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+$("#follow_form").html("<%= escape_javascript(render('users/follow')) %>");
+$("#followers").html('<%= @user.followers.count %>');
```
Comment (phamkykhoi) [inline app/views/relationships/destroy.js.erb]: ' ' => " "

File: app/views/relationships/destroy.js.erb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+$("#follow_form").html("<%= escape_javascript(render('users/follow')) %>");
```
Comment (phamkykhoi) [inline app/views/relationships/destroy.js.erb]: ' ' => " "

File: config/routes.rb [hunk 4]
```diff
@@ -7,5 +7,10 @@
   get     "login"     =>   "sessions#new"
   post    "login"     =>   "sessions#create"
   delete  "logout"    =>   "sessions#destroy"
-  resources :users
+  resources :users do
+    member do
```
Comment (dieunb) [inline config/routes.rb]: where is it?
```
+    member do
+      get :following, :followers
+    end
```

**Pull request merged**